# Open Science Toolbox for Animal Research

Open science has become a buzzword in the scientific community that too often misses the practical application for individual researchers. Transparent sharing of research results maximizes their utility and accelerate scientific progress. Open science practices support collaborations and improve communication between science and society.

The Open Science Toolbox for Animal Research was created by the Bf3R to provide an overview of open science tools that can be applied throughout the research process, from designing and conducting experiments to analyzing data and publishing results.

The tools listed in the table were selected based on their relevance to the animal research community in terms of their intended function to improve transparency and reproducibility. Inclusion of a tool does not imply endorsement by Bf3R, and exclusion of a tool is based solely on relevance; no assessments of tool quality were made or considered in the development of the Open Science Toolbox.

You can find more detailed information on the different open science practices in our corresponding article:
Diederich K, Schmitt K, Schwedhelm P, Bert B, Heinl C (2022) A guide to open science practices for animal research. PLOS Biology 20(9): e3001810. https://doi.org/10.1371/journal.pbio.3001810

We will follow the future developments of open science tools that are suitable for animal research and regularly update this toolbox. If you wish to suggest additional tools that should be added to the list, please email us at: OS-toolbox@bfr.bund.de.

The toolbox is also published on Zenodo: Diederich K, Schmitt K, Schwedhelm P, Bert B, Heinl C (2022) A guide to open science practices for animal research. PLOS Biology 20(9): e3001810. https://doi.org/10.1371/journal.pbio.3001810.


## Table

<table>
<colgroup>
<col style="width: 7%" />
<col style="width: 24%" />
<col style="width: 25%" />
<col style="width: 42%" />
</colgroup>
<thead>
<tr class="header">
<th></th>
<th>Open science practice</th>
<th>Specific Tools</th>
<th>Corresponding Links</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td rowspan="19"><blockquote>
<p>Planning</p>
</blockquote></td>
<td>Guidelines</td>
<td>PREPARE Guidelines</td>
<td><a
href="https://norecopa.no/prepare">https://norecopa.no/prepare</a></td>
</tr>
<tr class="even">
<td></td>
<td>UKRN Primers</td>
<td><a
href="https://www.ukrn.org/primers/">https://www.ukrn.org/primers/</a></td>
</tr>
<tr class="odd">
<td></td>
<td>ARRIVE Guidelines</td>
<td><a
href="https://arriveguidelines.org/">https://arriveguidelines.org/</a></td>
</tr>
<tr class="even">
<td>Design your study thoroughly</td>
<td>Literature and tools for the integration of sex and gender in
research</td>
<td><a
href="https://cihr-irsc.gc.ca/e/50836.html">https://cihr-irsc.gc.ca/e/50836.html</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Sample size calculator G*Power</td>
<td><a
href="https://www.psychologie.hhu.de/arbeitsgruppen/allgemeine-psychologie-und-arbeitspsychologie/gpower">https://www.psychologie.hhu.de/arbeitsgruppen/allgemeine-psychologie-und-arbeitspsychologie/gpower</a></td>
</tr>
<tr class="even">
<td></td>
<td>Software package for R for the statistical planning for animal
research</td>
<td><a href="https://invivostat.co.uk">https://invivostat.co.uk</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Tool for the randomization for experimental planning</td>
<td><a
href="https://www.randomizer.org/">https://www.randomizer.org/</a></td>
</tr>
<tr class="even">
<td></td>
<td>Creating a sharable scheme with the Experimental Design Assistant
(EDA)</td>
<td><a
href="https://eda.nc3rs.org.uk/">https://eda.nc3rs.org.uk/</a></td>
</tr>
<tr class="odd">
<td>Preregistration</td>
<td>Preclinicaltrials.eu</td>
<td><a
href="https://www.preclinicaltrials.eu/">https://www.preclinicaltrials.eu/</a></td>
</tr>
<tr class="even">
<td></td>
<td>Animalstudyregistry.org</td>
<td><a
href="https://www.animalstudyregistry.org">https://www.animalstudyregistry.org</a></td>
</tr>
<tr class="odd">
<td></td>
<td>OSF Registry</td>
<td><a
href="https://osf.io/registries">https://osf.io/registries</a></td>
</tr>
<tr class="even">
<td></td>
<td>List of journals offering registered reports</td>
<td><a
href="https://www.cos.io/initiatives/registered-reports">https://www.cos.io/initiatives/registered-reports</a></td>
</tr>
<tr class="odd">
<td>Writing a research data management plan</td>
<td>Research data management checklist from the Harvard Medical
School</td>
<td><a
href="https://datamanagement.hms.harvard.edu/plan-design">https://datamanagement.hms.harvard.edu/plan-design</a></td>
</tr>
<tr class="even">
<td></td>
<td>Research data management toolkit from JISC</td>
<td><a
href="https://www.jisc.ac.uk/full-guide/rdm-toolkit">https://www.jisc.ac.uk/full-guide/rdm-toolkit</a></td>
</tr>
<tr class="odd">
<td></td>
<td>DMPTool</td>
<td>https://dmptool.org/</td>
</tr>
<tr class="even">
<td></td>
<td>DMPonline</td>
<td><a
href="https://dmponline.dcc.ac.uk/">https://dmponline.dcc.ac.uk/</a></td>
</tr>
<tr class="odd">
<td></td>
<td>ARGOS</td>
<td><a
href="https://argos.openaire.eu">https://argos.openaire.eu</a></td>
</tr>
<tr class="even">
<td>Writing a non-technical summary</td>
<td>Alures: the Europe-wide NTS database</td>
<td><a
href="https://webgate.ec.europa.eu/envdataportal/web/resources/alures/submission/nts/list">https://webgate.ec.europa.eu/envdataportal/web/resources/alures/submission/nts/list</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Guide to writing non-technical summaries from UAR</td>
<td><a
href="https://concordatopenness.org.uk/guide-to-writing-non-technical-summaries">https://concordatopenness.org.uk/guide-to-writing-non-technical-summaries</a></td>
</tr>
<tr class="even">
<td rowspan="7"><blockquote>
<p>CONDUCTING EXPERIMENTS</p>
</blockquote></td>
<td>Using an electronic lab notebook</td>
<td>Table of ELNs with features</td>
<td><a
href="https://zenodo.org/record/4723753">https://zenodo.org/record/4723753</a></td>
</tr>
<tr class="odd">
<td>Sharing protocols</td>
<td>Protocols.io</td>
<td><a
href="https://www.protocols.io/">https://www.protocols.io/</a></td>
</tr>
<tr class="even">
<td></td>
<td>Protocol exchange</td>
<td><a
href="https://protocolexchange.researchsquare.com/">https://protocolexchange.researchsquare.com/</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Bio-protocol</td>
<td><a
href="https://bio-protocol.org/Default.aspx">https://bio-protocol.org/Default.aspx</a></td>
</tr>
<tr class="even">
<td>Reporting critical incidents</td>
<td>Critical incident reporting CIRS-LAS</td>
<td><a
href="https://www.cirs-las.de/home">https://www.cirs-las.de/home</a></td>
</tr>
<tr class="odd">
<td>Sharing animals, organs and tissue</td>
<td>Online sharing platform for organs and tissues</td>
<td><a href="http://www.animatch.eu">www.animatch.eu</a></td>
</tr>
<tr class="even">
<td></td>
<td>Searchable online data base of mouse strain resources from multiple
repositories</td>
<td><a
href="http://www.findmice.org/index">http://www.findmice.org/index</a></td>
</tr>
<tr class="odd">
<td rowspan="7"><blockquote>
<p>ANALYSIS</p>
</blockquote></td>
<td>Writing transparent code</td>
<td>Jupyter Notebooks</td>
<td><a href="https://jupyter.org/">https://jupyter.org/</a></td>
</tr>
<tr class="even">
<td></td>
<td>GitHub</td>
<td><a href="https://github.com/">https://github.com/</a></td>
</tr>
</tr>
<tr class="even">
<td></td>
<td>GitLab</td>
<td><a href="https://gitlab.com/">https://gitlab.com/</a></td>
</tr>
<tr class="odd">
<td></td>
<td>R</td>
<td><a
href="https://www.r-project.org/">https://www.r-project.org/</a></td>
</tr>
<tr class="even">
<td></td>
<td>Quarto</td>
<td><a href="https://quarto.org/">https://quarto.org/</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Codeberg</td>
<td><a href="https://codeberg.org/">https://codeberg.org/</a></td>
</tr>
<tr class="even">
<td>Choosing transparent data visualization</td>
<td>Paper with a list of free tools for more transparent data
visualization</td>
<td><a
href="https://www.ahajournals.org/doi/epub/10.1161/CIRCULATIONAHA.118.037777">https://www.ahajournals.org/doi/epub/10.1161/CIRCULATIONAHA.118.037777</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Tool to check graph accessibility for color blind persons</td>
<td><a href="https://colororacle.org/index.html">https://colororacle.org/index.html</a></td>
</tr>
<tr class="even">
<td rowspan="29"><blockquote>
<p>Publication</p>
</blockquote></td>
<td>Adopting the FAIR data principles</td>
<td>A guide on how to implement the FAIR data principles</td>
<td><a
href="https://www.go-fair.org/how-to-go-fair/">https://www.go-fair.org/how-to-go-fair/</a></td>
</tr>
<tr class="odd">
<td rowspan="2">Using field specific reporting guidelines</td>
<td>ARRIVE Guidelines</td>
<td><a
href="https://arriveguidelines.org/">https://arriveguidelines.org/</a></td>
</tr>
<tr class="even">
<td>Tool from the EQUATOR Network to find specific reporting
guidelines</td>
<td><a
href="https://www.equator-network.org/reporting-guidelines/">https://www.equator-network.org/reporting-guidelines/</a></td>
</tr>
<tr class="odd">
<td></td>
<td>MERIDIAN â€“ collection of all reporting guidelines involving
animals</td>
<td><a
href="https://meridian.cvm.iastate.edu/">https://meridian.cvm.iastate.edu/</a></td>
</tr>
<tr class="even">
<td>Using persistent identifiers</td>
<td>Unmistakably identify publications: Digital Object Identifier
(DOI)</td>
<td><a href="https://www.doi.org/">https://www.doi.org/</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Unmistakably identify authors: ORCID-ID</td>
<td><a href="https://orcid.org/">https://orcid.org/</a></td>
</tr>
<tr class="even">
<td></td>
<td>Unmistakably identify resources: Research Resource Identifiers
(RRID)</td>
<td><a
href="https://scicrunch.org/resources">https://scicrunch.org/resources</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Unmistakably identify mouse lines: The Mouse Genome Informatics
(MGI)</td>
<td><a
href="http://www.informatics.jax.org/mgihome/nomen/index.shtml">http://www.informatics.jax.org/mgihome/nomen/index.shtml</a></td>
</tr>
<tr class="even">
<td>Publishing preprints</td>
<td>Searchable database of preprint servers</td>
<td><a
href="https://asapbio.org/preprint-servers">https://asapbio.org/preprint-servers</a></td>
</tr>
<tr class="odd">
<td></td>
<td>BioRxiv</td>
<td><a href="https://www.biorxiv.org/">https://www.biorxiv.org/</a></td>
</tr>
<tr class="even">
<td></td>
<td>MedRxiv</td>
<td><a href="https://www.medrxiv.org/">https://www.medrxiv.org/</a></td>
</tr>
<tr class="odd">
<td></td>
<td>OSF preprints</td>
<td><a
href="https://osf.io/preprints/">https://osf.io/preprints/</a></td>
</tr>
<tr class="even">
<td>Publishing negative results</td>
<td>fiddle- file drawer data liberation effort to identify a way of
publication for null results</td>
<td><a
href="https://s-quest.bihealth.org/fiddle/">https://s-quest.bihealth.org/fiddle/</a></td>
</tr>
<tr class="odd">
<td>Publishing open access</td>
<td>Journals listed by the Directory of Open Access Journals (DOAJ)</td>
<td><a href="https://doaj.org/">https://doaj.org/</a></td>
</tr>
<tr class="even">
<td></td>
<td>Gold Open Access: List of open access biomedical journals</td>
<td><a
href="https://s-quest.bihealth.org/OAPositiveList/">https://s-quest.bihealth.org/OAPositiveList/</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Green Open Access: List of open Access repositories OpenDOAR :</td>
<td><a
href="https://v2.sherpa.ac.uk/cgi/search/repository/advanced">https://v2.sherpa.ac.uk/cgi/search/repository/advanced</a></td>
</tr>
<tr class="even">
<td></td>
<td>Find open access policies of journals and publishers: Sherpa
Romeo</td>
<td><a
href="https://v2.sherpa.ac.uk/romeo/search.html">https://v2.sherpa.ac.uk/romeo/search.html</a></td>
</tr>
<tr class="odd">
<td>Depositing code and data in public repositories</td>
<td>Finding a research field specific repository:</td>
<td><p><a
href="https://www.re3data.org/browse/by-subject/">https://www.re3data.org/browse/by-subject/</a></p>
<p><a
href="https://beta.fairsharing.org/">https://beta.fairsharing.org/</a></p></td>
</tr>
<tr class="even">
<td></td>
<td>Open Science Framework</td>
<td><a href="https://osf.io/">https://osf.io/</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Figshare</td>
<td><a href="https://figshare.com/">https://figshare.com/</a></td>
</tr>
<tr class="even">
<td></td>
<td>Dryad</td>
<td><a
href="https://datadryad.org/stash">https://datadryad.org/stash</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Zenodo</td>
<td><a href="https://zenodo.org/">https://zenodo.org/</a></td>
</tr>
<tr class="even">
<td>Attributing creative commons licenses</td>
<td>Attributing the adequate creative commons license</td>
<td><a
href="https://creativecommons.org/choose/">https://creativecommons.org/choose/</a></td>
</tr>
<tr class="odd">
<td>Giving author/ contributor information</td>
<td>CRediT (Contributor Roles Taxonomy)</td>
<td><a href="https://credit.niso.org/">https://credit.niso.org/</a></td>
</tr>
<tr class="even">
<td></td>
<td>Tenzing</td>
<td><a href="http://www.tenzing.club">http://www.tenzing.club</a></td>
</tr>
<tr class="odd">
<td>Publishing and connecting all outcomes</td>
<td>Open Science Framework</td>
<td><a href="https://osf.io/">https://osf.io/</a></td>
</tr>
<tr class="even">
<td>Communicating research</td>
<td>Mastodon</td>
<td><a href="https://mastodon.social/">https://mastodon.social/</a></td>
</tr>
<tr class="odd">
<td></td>
<td>Twitter</td>
<td><a href="https://twitter.com/">https://twitter.com/</a></td>
</tr>
<tr class="even">
<td></td>
<td>ResearchGate</td>
<td><a
href="https://www.researchgate.net/">https://www.researchgate.net/</a></td>
</tr>
</tbody>
</table>
